docs/source/asr/configs.rst

```diff
@@ -416,3 +431,15 @@ A Citrinet-512 config might look like below.
         se_context_size: ${model.model_defaults.se_context_size}
 
 As discussed above, Citrinet uses the ``ConvASRDecoder`` as the decoder layer similar to QuartzNet. Only the configuration must be changed slightly as Citrinet is utilizes sub-word tokenization.
+
+
+Conformer-CTC
+~~~~~~~~~~~~~
+
+You may find the config files for Conformer-CTC model with character-based encoding and word-piece encoding at ``<NeMo_git_root>/examples/asr/conf/conformer/conformer_ctc_char.yaml`` and ``<NeMo_git_root>/examples/asr/conf/conformer/conformer_ctc_bpe.yaml`` respectively.
+Some components of the configs of `Conformer-CTC <./models.html#Conformer-CTC>`__ including
+datasets (train_ds, validation_ds, and test_ds), opimizer (optim), augmentation (spec_augment), decoder, trainer, and exp_manager are
+similar to other ASR models like `QuartzNet <./models.html#QuartzNet>`__. There should be a tokenizer section which you may specify the tokenizer if you want to use word-piece encoding instead of character-based encoding.
```

Comment:
Not just word pieces. Rename to sub-word encoding.

Reply:
done.